cl/audio/api_views.py

```diff
@@ -15,6 +15,16 @@ class AudioViewSet(LoggingMixin, viewsets.ModelViewSet):
         "date_modified",
         "date_blocked",
     )
+    # Default cursor ordering key
+    ordering = "-id"
+    # Other cursor ordering keys
+    other_cursor_ordering_keys = [
+        "id",
+        "date_created",
+        "-date_created",
+        "date_modified",
+        "-date_modified",
+    ]
```

Comment:
I think it might make more sense to call these `cursor_ordering_fields`, and to include any of the regular ordering keys in the list as well.

Maybe, also, `default_ordering`? 

So:

```suggestion
    default_ordering = "-id"
    # Other cursor ordering keys
    cursor_ordering_fields = [
        "id",
        "-id",
        "date_created",
        "-date_created",
        "date_modified",
        "-date_modified",
    ]
```

Calling it `fields` instead of `keys` makes it consistent with the DRF attribute, `ordering_fields`.

Reply:
Sure I renamed `other_cursor_ordering_keys` to `cursor_ordering_fields` for clarity. 
I also refactored the code so that declaring the inverse fields like `"-id" and "-date_created"` is no longer required. Similar to `ordering_fields`, you only need to specify the base fields like `"id"` and `"date_created"`.

I kept `ordering` because it's the original property name required by the DRF `CursorPagination` class.